### PR TITLE
Update the Linux documentation

### PR DESCRIPTION
**Pull request by darcywong00**
For #20, incorporate the man pages from https://manpages.debian.org/unstable/keyman/

* Moved installing-keyboard.php as a guides
* Update common questions from keyman.com

### CODE REVIEW

File: products/linux/11.0/web.config [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mcdurdin) [inline products/linux/11.0/web.config]: Is this file needed given we have web.config in the parent folder which does the same things?

Comment (darcywong00) [reply products/linux/11.0/web.config]: I lifted it from `developer\engine\web\11.0`.

It seems like it's needed? Without it, I couldn't browse to /products/linux/11.0/common

Comment (mcdurdin) [reply products/linux/11.0/web.config]: Okay, it's no big deal anyway :)
